packages/api/src/external/carequality/command/cq-directory/rebuild-cq-directory.ts

```diff
@@ -16,13 +18,12 @@ dayjs.extend(duration);
 const BATCH_SIZE = 1000;
 const SLEEP_TIME = dayjs.duration({ milliseconds: 750 });
 
-const sqlDBCreds = Config.getDBCreds();
-const dbCreds = JSON.parse(sqlDBCreds);
-
-const sequelize = new Sequelize(dbCreds.dbname, dbCreds.username, dbCreds.password, {
-  host: dbCreds.host,
-  port: dbCreds.port,
-  dialect: dbCreds.engine,
+const dbCreds = Config.getDBCreds();
+const sequelize = initDBPool(dbCreds, {
+  max: 10,
+  min: 1,
+  acquire: 30000,
+  idle: 10000,
 });
```

Comment:
Recommending we don't initialize Sequelize/DB pool directly - and be specific about some configs like DB conn pool.